### PR TITLE
Bot keyboards

### PR DESCRIPTION
**Pull request by Baxodir0327**
please Nodir bro and Doniyor check this code

### CODE REVIEW

**Review by EnU098 — APPROVED** [review]
ok good

File: src/main/java/com/company/client/bot/MyBot.java [hunk 1]
```diff
@@ -1,26 +1,122 @@
...
+                startMainPage(chatId,createButtonService);
+                userService.update(user);
             }
+
+        } else if (update.hasCallbackQuery()) {
+
+        }
+    }
+
+    private void startMainPage(Long chatId, CreateButtonService createButtonService) {
```
Comment (j-onfroy) [inline src/main/java/com/company/client/bot/MyBot.java:90]: change method name to mainPage

**Review by j-onfroy — APPROVED** [review]

**Review by Akmal-85 — APPROVED** [review]
ok

File: src/main/java/com/company/client/bot/MyBot.java (hunk 1 above)
Comment (Baxodir0327) [reply src/main/java/com/company/client/bot/MyBot.java:90]: ok

**Review by Akmal-85 — APPROVED** [review]
ok

**Review by Akmal-85 — APPROVED** [review]
ok

**Pull request merged**